docs/_articles/en/user-guide/detect-conflicts.md

```diff
@@ -1,33 +1,32 @@
 ---
-title: Detect Conflicts
+title: Detect Conflicts on Deploy
 lang: en
 ---
 
-Whenever you retrieve or deploy source to keep the local project and your default org in sync, the operation could detect conflicts. The conflict detection feature is available only in non-scratch orgs such as sandboxes or dev orgs.
+When you deploy source to your default org, the operation could detect conflicts on deploy to help you avoid accidentally overwriting others’ changes in the Org. The conflict detection feature is available for any Org that is not using source tracking, which is most non-scratch orgs such as sandboxes or dev orgs.
```

Comment:
Because we're heavily pushing source tracking in "Developer" sandboxes for DevOps Center, let's think about how we can clarify. This is tricky.

Normally, I was taught to go positive vs. negative. What you can do vs. what you can't. But in this case it may be easier and simpler.

The conflict detection feature isn't available in orgs that use source tracking, such as scratch orgs and Developer sandboxes. 

Also, we made a choice at some point in SFDX doc pod to NOT capitalize "org".

Reply:
changed Org to org. Leaving the rest as it based on slack convo.